wafer/schedule/templates/wafer.schedule/current.html

```diff
@@ -36,13 +40,18 @@ <h1>{% trans "Happening now" %}</h1>
               {% if item.item == "unavailable" %}
                 <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="unavailable"></td>
               {% else %}
-                {% if item.note == "complete" %}
-                  <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="completed">
-                {% elif item.note == "current" %}
-                  <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="current_active">
-                {% else %}
-                  <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="future">
-                {% endif %}
+                <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}"
+                  {% if item.item.venue.pk == highlight_venue_pk %}
+                    class="schedule-highlight-venue"
```

Comment:
Could this end up with the class be specified multiple times?

Reply:
I see from the tests that is does. I would have thought that was not okay in HTML?

Reply:
I misremembered the HTML spec on this - fixed